geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestBase.java

```diff
@@ -223,7 +220,7 @@ public void run2() throws CacheException {
   }
 
   private static String printMap(Map<String, Object[]> m) {
-    Iterator<Map.Entry<String, Object[]>> itr = m.entrySet().iterator();
+    java.util.Iterator<java.util.Map.Entry<String, Object[]>> itr = m.entrySet().iterator();
```

Comment:
Why do Iterator and Map require fully qualified paths?

Reply:
Good catch, something went wrong in auto-formatting.